lib/ansible/modules/storage/netapp/na_cdot_user.py

```diff
@@ -204,6 +204,9 @@ def get_user(self):
             # Error 16034 denotes a user not being found.
             if str(e.code) == "16034":
                 return False
+            # Error 16043 denotes the user existing, but the application missing
+            elif str(e.code) == "16043":
+                return False
             else:
                 self.module.fail_json(msg='Error getting user %s' % self.name, exception=str(e))
```

Comment:
These should not be using the exception key like this.  exception is only for the traceback.  So this line should really look like this:
``` python
import traceback

from ansible.module_utils._text import to_native
[...]

  self.module.fail_json(msg='Error getting user %s: %s' % (self.name, to_native(e)), exception=traceback.format_exc(e))
```